services/distribution/src/main/java/app/coronawarn/server/services/distribution/dgc/client/ProdDigitalCovidCertificateClient.java

```diff
@@ -83,6 +85,9 @@ public ProdDigitalCovidCertificateClient(DigitalCovidCertificateFeignClient digi
     logger.debug("Get " + fetchEntityName + " from DCC");
     try {
       ResponseEntity<T> response = responseSupplier.get();
+
+      logger.info(fetchEntityName + "has been fetched and contains the following data: "
+          + stringifyObject(response.getBody()));
```

Comment:
can this result again in an IOException (stream already closed) or something?

Reply:
yes, it can. I mean,  [Response execute(Request request, Options options)] only throws IOException. Even if the signature fails or any other reason. But I don't think it's possible to throw stream already closed anymore.